runtime/amd64.S

```diff
@@ -679,7 +679,7 @@ G(caml_system__frametable):
 #elif defined(SYS_mingw64) || defined(SYS_cygwin)
         .section .rdata,"dr"
 #else
-        .section    .rodata.cst8,"a",@progbits
+        .section    .rodata.cst16,"aM",@progbits,16
 #endif
         .globl  G(caml_negf_mask)
         .align  SIXTEEN_ALIGN
```

Comment:
@jacobly0 Is it the case that this alignment directive (and the one below) are no longer needed?

Reply:
At least one `.align` directive is needed per section, as the largest alignment that appears in a section defines the alignment of the entire section, and the default is 1 byte alignment.  The fourth argument to a `.section` directive (with `"M"` flag but not `"S"` flag) defines how big each entry in the section is, and is independent of alignment (As a random example, ELF64 `.rela.*` sections have an alignment of 8 bytes but an entsize of 24 bytes).  This means that only the second `.align` directive is superfluous here, since there happens to be exactly 16 bytes between it and the previous one.  Since you brought up alignment, I also added a missing one in the other changed file.